### PR TITLE
FortyTwo users / connections cleanup

### PR DESCRIPTION
**Pull request by leogrim**
@andrewconner @stkem

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/admin/AdminUserController.scala [hunk 1]
```diff
@@ -715,4 +716,48 @@ class AdminUserController @Inject() (
...
+      }
+    }
+
+    if (!readOnly) {
+      toBeCreated.foreach { sui =>
+        val currentHasher = Registry.hashers.currentHasher
+        val pInfo = currentHasher.hash(ExternalId[Nothing]().toString)
+        authCommander.saveUserPasswordIdentity(
+          userIdOpt = Some(sui.userId.get),
+          identityOpt = sui.credentials,
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/admin/AdminUserController.scala]: Not sure if this part is right. It's using the credentials of the other network?

File: kifi-backend/modules/shoebox/app/com/keepit/admin/AdminUserController.scala [hunk 2]
```diff
@@ -715,4 +716,48 @@ class AdminUserController @Inject() (
...
+          email = sui.credentials.get.email.get,
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/admin/AdminUserController.scala]: Should use user's primary email.

File: kifi-backend/modules/shoebox/app/com/keepit/admin/AdminUserController.scala [hunk 3]
```diff
@@ -715,4 +716,48 @@ class AdminUserController @Inject() (
...
+          passwordInfo = pInfo,
+          firstName = sui.credentials.get.firstName,
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/admin/AdminUserController.scala]: Should use user's first / last name

File: kifi-backend/modules/shoebox/app/com/keepit/admin/AdminUserController.scala [hunk 4]
```diff
@@ -715,4 +716,44 @@ class AdminUserController @Inject() (
...
+          passwordInfo = pInfo,
+          firstName = user.firstName,
+          lastName = user.lastName,
+          isComplete = true
+        )
+      }
+    }
+    Ok(Json.toJson(problematicUsers))
+  }}
+
```
Comment (leogrim) [inline kifi-backend/modules/shoebox/app/com/keepit/admin/AdminUserController.scala:741]: @andrewconner merging

**Pull request merged**